sycl/test-e2e/DeviceGlobal/device_global_static.cpp

```diff
@@ -3,7 +3,10 @@
 //
 // The OpenCL GPU backends do not currently support device_global backend
 // calls.
-// UNSUPPORTED: opencl && gpu
+//
+// FIXME: make this work for HIP
+//
+// UNSUPPORTED: hip_amd, opencl && gpu
```

Comment:
```suggestion
// UNSUPPORTED: opencl && gpu
//
// See ADD_GH_ISSUE_LINK_HERE
// UNSUPPORTED: hip_amd
```
where `ADD_GH_ISSUE_LINK_HERE` would be the link to a Github issue detailing the failure.

Reply:
Thanks @steffenlarsen have done so.